app/store/modules/product/index.js

```diff
@@ -0,0 +1,22 @@
+// store/modules/product/index.js
+
+import product from './mock';
+
+export default {
+  namespaced: true,
+  state: {
+    items: product // demo
```

Comment:
- [ ] Please also rename property to `product` (variable name `productDate` would have been ok).

Reply:
Ok, changed property to `product` and the import variable to 'productData'.